tools/requirements.txt

```diff
@@ -1,2 +1,2 @@
 Markdown==3.1.1
-lxml==4.6.5
+lxml==4.4.2
```

Comment:
This looks like a mistake in the merge, as it basically would revert https://github.com/digidotcom/xbee-micropython/pull/77